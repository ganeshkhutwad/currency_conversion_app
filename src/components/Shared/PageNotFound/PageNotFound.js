import React from 'react';

const PageNotFound = () => {
    return (
        <div>
            <h3>Page Not Found</h3>
            <p>This page is not available.</p>
        </div>
    );
};

export default PageNotFound;
