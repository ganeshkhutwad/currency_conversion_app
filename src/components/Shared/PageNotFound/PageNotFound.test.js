import React from 'react';
import { render } from '@testing-library/react';
import PageNotFound from './PageNotFound';

test('Snapshot For PageNotFound Component', () => {
    const { asFragment } = render(<PageNotFound />);
    expect(asFragment()).toMatchSnapshot();
});
