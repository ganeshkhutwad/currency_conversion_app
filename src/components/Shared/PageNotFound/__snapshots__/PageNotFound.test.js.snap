// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot For PageNotFound Component 1`] = `
<DocumentFragment>
  <div>
    <h3>
      Page Not Found
    </h3>
    <p>
      This page is not available.
    </p>
  </div>
</DocumentFragment>
`;
